cartography/intel/aws/iam.py

```diff
@@ -925,6 +929,57 @@ def sync_user_access_keys(
     )
 
 
+def _set_used_state_tx(
+    tx: neo4j.Transaction, project_id: str, common_job_parameters: Dict, update_tag: int,
+) -> None:
+    ingest_role_used = """
+    MATCH (n:AWSRole)<-[:RESOURCE]-(:AWSAccount{id: {AWS_ID}})
```

Comment:
same as other clouds, let's flip this query. start with CloudanixWorkspace->.... AWSRole